API/Models/GetQuestionDTO.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Newtonsoft.Json;
```

Comment:
Why did you create a new class instead of just updating https://github.com/KamandaX/RecommendationAPI/blob/master/API/Models/Question.cs ?

Reply:
This class is used specifically for this endpoint with all irrelevant data stripped. As main class has some additional parameters contract does not require, an additional class is used.

Reply:
Have you considered hiding that irrelevant data with [JsonIgnore]?

Reply:
Yes. While it might technically work in this case, consider if we added another route that also queries the questions but provides different properties. That would require yet another set of rules. Instead, it is generally bettter to have a DTO that queries database and another DTO for route.
Also, it is probably better not to use JSON serialization on DTO that is associated woth the db unless we absolutely use all the parameters.

Reply:
If the idea is to have multiple DTOs for different routes then perhaps we could organize them into folders?

Reply:
That would be a good idea, not on the scope of this MR, though.